firmware/controllers/lua/lua_hooks.cpp

```diff
@@ -616,7 +616,7 @@ void configureRusefiLuaHooks(lua_State* l) {
 	lua_register(l, "findCurveIndex", [](lua_State* l) {
 		auto name = luaL_checklstring(l, 1, nullptr);
 		auto result = getCurveIndexByName(name);
-		if (result == EFI_ERROR_CODE) {
+		if (result == (int) EFI_ERROR_CODE) {
```

Comment:
EFI_ERROR_CODE is a hack at best, we should probably remove it all together and use `expected<T>` instead.